cves/CVE-2011-3923.yml

```diff
@@ -150,13 +142,16 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    Between the VCCS and the fix there was typical development and maintenance on
```

Comment:
Lots of incomplete/ weak sentence structure here. The second sentence starting with "Things" doesn't really contain a subject at the beginning of the sentence, which could hinder readability.

Reply:
I'll be sure to give this entry another pass and fix any lingering odd sentences. Thanks for the feedback!